### PR TITLE
chore: CodeQL Action

### PR DESCRIPTION
**Pull request by naveensrinivasan**
>This action runs GitHub's industry-leading semantic code analysis engine, CodeQL, against a repository's source code to find security vulnerabilities. It then automatically uploads the results to GitHub so they can be displayed in the repository's security tab. CodeQL runs an extensible set of queries, which have been developed by the community and the GitHub Security Lab to find common vulnerabilities in your code.

scorecards recommendation   https://github.com/ossf/scorecard/blob/main/docs/checks.md#sast https://github.com/angular/components/blob/653457eaf48faab99227f37bc2fe104d9f308787/.github/workflows/scorecard.yml#L34

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@devversion 👀

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    #        ┌───────────── minute (0 - 59)
+    #        │  ┌───────────── hour (0 - 23)
+    #        │  │ ┌───────────── day of the month (1 - 31)
+    #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │  │ │ │ │
+    #        │  │ │ │ │
+    #        │  │ │ │ │
+    #        *  * * * *
+    - cron: '30 1 * * 0'
```
Comment (devversion) [inline .github/workflows/codeql.yml:19]: Can we just have a comment saying: "# Cronjob to run every Sunday at 01:30`? (instead of the large one)

**Review by devversion — APPROVED** [review]
LGTM overall, but it's unclear to me if we want to activate this. @josephperrott enabled other similar actions, so he should know.

**Review by josephperrott — APPROVED** [review]
LGTM

We can enable it and see if we find value from it.

**Comment by naveensrinivasan** [comment]
> LGTM
> 
> We can enable it and see if we find value from it.

Do you think anything else from me to merge this?

**Comment by devversion** [comment]
I think you just need to format the file, lint is failing. I can add merge ready then (to indicate that this should be merged)

**Comment by naveensrinivasan** [comment]
> I think you just need to format the file, lint is failing. I can add merge ready then (to indicate that this should be merged)

Can you please point me to the tool to fix this?

**Comment by naveensrinivasan** [comment]
I am going to close this for now.  Thanks

**Pull request closed without merging**